app/models/company.rb

```diff
@@ -1,2 +1,7 @@
 class Company < ActiveRecord::Base
+  validates :name,            presence: true
+  validates :description,     presence: true
+  validates :sub_description, presence: true
```

Comment:
@arafatm This is a tag line from the company's description, i.e
- 3sixD Consulting => description for a company
- We create openings - Post your cvs - Get hired => sub_description